deepchecks/nlp/utils/text_properties.py

```diff
@@ -204,7 +211,9 @@ def formality(raw_text: Sequence[str], device: Optional[int] = None) -> List[flo
     {'name': 'Subjectivity', 'method': subjectivity, 'output_type': 'numeric'},
     {'name': 'Toxicity', 'method': toxicity, 'output_type': 'numeric'},
     {'name': 'Fluency', 'method': fluency, 'output_type': 'numeric'},
-    {'name': 'Formality', 'method': formality, 'output_type': 'numeric'}
+    {'name': 'Formality', 'method': formality, 'output_type': 'numeric'},
+    {'name': 'Lexical Density', 'method': lexical_density, 'output_type': 'numeric'},
+    {'name': 'Noun Count', 'method': noun_count, 'output_type': 'numeric'},
```

Comment:
We discussed doing Unique Noun Count, I think it will be better :)

Reply:
Yeah, my bad, actually the logic is written to calculate the unique noun counts only, but the naming is just Noun Count. I will change the name to Unique Noun Count!!